src/index.ts

```diff
@@ -2,6 +2,8 @@ import bindings = require('bindings');
 
 const mc = bindings('mongocrypt');
 
+export { cryptoCallbacks } from './crypto_callbacks';
```

Comment:
Given that these are now part of the `src/` tree anyway (which they should be imo), should we also just use them by default so we can remove them from the driver eventually (I would say so)? e.g.

```js
import { cryptoCallbacks } from './crypto_callbacks';
export { cryptoCallbacks };

// ...

export const MongoCrypt: MongoCryptConstructor = class MongoCrypt extends (mc.MongoCrypt as MongoCryptConstructor) {
  constructor(options: ConstructorParameters<MongoCryptConstructor>[0]) {
    super({ cryptoCallbacks, ...options });
  }
}
```

Reply:
Yes, thank you for suggesting. I can make this improvement. I was thinking about how we could do this, I was inclined to avoid subclassing.. for no real reason, and I was caught up in thinking how I could statically assign it to the constructor or some other strange communication mechanism to get it over to the bindings. Regardless, happy with this solution, and I can do a bit of TS improvement to make this read nicely.